### PR TITLE
bsunanda:Run2 hcx17 

### PR DESCRIPTION
**Pull request by bsunanda**
Managed to take the previous pull request. Merged with Yana's changes after rebasing and committing the version

### CODE REVIEW

**Comment by ianna** [comment]
+1

File: Geometry/Records/src/ES_MuonDDDConstants.cc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#include "Geometry/MuonNumbering/interface/MuonDDDConstants.h"
```
Comment (ianna) [inline Geometry/Records/src/ES_MuonDDDConstants.cc]: @cmsbuild - it is very strange that this file is not in IB. Any idea what's wrong?

Comment (davidlange6) [inline Geometry/Records/src/ES_MuonDDDConstants.cc]: https://github.com/cms-sw/cmssw/blob/68b4ccfc9d6f7b0ae0980969622686957e61fc9a/Geometry/MuonNumbering/interface/MuonDDDConstants.h
?

Comment (bsunanda) [inline Geometry/Records/src/ES_MuonDDDConstants.cc]: Well Geometry/MuonNumbering/interface/MuonDDDConstants.h is there in the IB  on top of which this branch was made - it is made on top of  CMSSW_7_6_X_2015-06-30-2300. I did not touch anything about Muon here. I removed the merge conflict and made a new tag.
________________________________

**Comment by bsunanda** [comment]
I dont think the git-branch has anything to do with the failure. I tried runTheMatrix.py -l 25202 and it ended after going through all the steps. Recently lxplus is behaving badly with many intermitent network issues. Please retry the tests. The changes here will not affect any workflow at the moment because their connection to SIM-DIGI-RECO is still to come.
________________________________

**Comment by bsunanda** [comment]
Just adding the results:
25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25+MINIAODMCUP15 Step0-PASSED Step1-PASSED Step2-PASSED Step3-PASSED Step4-PASSED  - time date Thu Jul  2 08:29:56 2015-date Thu Jul  2 07:33:43 2015; exit: 0 0 0 0 0
1 1 1 1 1 tests passed, 0 0 0 0 0 failed

________________________________
From: Sunanda Banerjee
Sent: 02 July 2015 08:16
To: cms-sw/cmssw; cms-sw/cmssw
Subject: RE: [cmssw] bsunanda:Run2 hcx17 (#9994)

I dont think the git-branch has anything to do with the failure. I tried runTheMatrix.py -l 25202 and it ended after going through all the steps. Recently lxplus is behaving badly with many intermitent network issues. Please retry the tests. The changes here will not affect any workflow at the moment because their connection to SIM-DIGI-RECO is still to come.
________________________________

**Comment by ianna** [comment]
@bsunanda - I'm afraid you've missed a few changes from my branch which was merged into your previous one. Would you like me to re-submit it?

**Comment by bsunanda** [comment]
I tried to do the merge and resolved the issues. When there was a merge conflict it really gave many problems and it might have left these problems. Could you send me the addition in a branch and let me merge them and make the update. Best regards           Sunanda
________________________________

**Comment by ianna** [comment]
@bsunanda - this is the branch:
https://github.com/bsunanda/cmssw/pull/5/files

**Comment by bsunanda** [comment]
Sorry for missing out the changes in CondCore/Utilities for the previous commit
________________________________

**Comment by ianna** [comment]
@bsunanda - thanks! I'm preparing new payloads - will send you a config for tests by e-mail.

**Comment by ianna** [comment]
+1

**Comment by ianna** [comment]
@cmsbuild please test

**Comment by ianna** [comment]
@ggovi and @diguida - this PR contains new schema for HcalParameters. Please, update DropBox. Thanks.

**Comment by ianna** [comment]
@degano and @smuzaffar - it seems everything is on hold - labels are not updating, tests are not triggered and not just for this PR.

**Comment by bsunanda** [comment]
As I mentioned earlier I ran this test on the build from which I made the tag and it passed. I sent the results and the diagnostics has no resemblance to any changes made in the committed code.
________________________________

**Comment by davidlange6** [comment]
@cmsbuild, please test

**Comment by slava77** [comment]
@smuzaffar jenkins still builds against CMSSW_7_6_X_2015-07-04-2300
https://cmssdt.cern.ch/jenkins/job/ib-any-integration/5991/console

**Comment by ianna** [comment]
@cmsbuild please test

**Comment by ianna** [comment]
+1

**Comment by ianna** [comment]
@ggovi, @diguida, @mmusich - this PR contains a new Hcal parameters schema. Please, use it to update DropBox. Thanks.

**Comment by ggovi** [comment]
+1

**Pull request closed without merging**